### ISSUE TITLE
Module not found: Can't resolve 'dom-helpers/events/off'

### ISSUE THREAD

**Issue by Ich-bin-naoya** [oliviertassinari/react-swipeable-views#567]
- [ ] I have searched the [issues](https://github.com/oliviertassinari/react-swipeable-views/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
When building with @material-ui I receive a module not fond error for dom-helpers.
And I did install  the latest dom-helpers@5.1.3 to solve this. But It did not work.

## Your Environment

| Tech                   | Version |
|------------------------|---------|
| react-swipeable-views  |     0.13.4    |
| React                  |      16.12.0   |
|     material-ui     |   v4     |
| nextjs              |    9.1.7     |

Is this bug or my fault? :/

**Comment by brunocrosier** [comment]
check #566 and #563 

the current release (0.13.4) is broken

change your version of "react-swipeable-views" to 0.13.3 for now and it will work

**Comment by estubmo** [comment]
Having the same issue.

**Comment by repl-sean-heintz** [comment]
I opened a PR which fixed this. It's been merged, just waiting on the maintainer to cut a new release.

**Comment by Ich-bin-naoya** [comment]
> check #566 and #563
> 
> the current release (0.13.4) is broken
> 
> change your version of "react-swipeable-views" to 0.13.3 for now and it will woｖrk

v0.13.3 is worked !!!

Thank you!!

**Comment by NawarA** [comment]
I see this issue:

```Could not find module in path: 'dom-helpers/transition/properties' relative to '/node_modules/react-swipeable-views/lib/SwipeableViews.js'```

Same issue or a different bug?

**Comment by jinsley8** [comment]
I'm also getting this with the latest 0.13.6:

`Could not find module in path: 'dom-helpers/transition/properties' relative to '/node_modules/react-swipeable-views/lib/SwipeableViews.js'`

**Comment by tinder-rhsiao** [comment]
https://github.com/oliviertassinari/react-swipeable-views/pull/565#discussion_r367253438 0.13.6 broken with dom-helpers breaking changes

**Comment by oliviertassinari** [comment]
Duplicate of #564
